### DISCUSSION TITLE
Problem in linking a PR with a task in Github Project

### DISCUSSION THREAD

**Discussion by rikhiltaneja** [community/community#122107] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

I'm curious if it's possible to link a Pull Request (PR) with a task created within a GitHub Project. While I've managed to manually link tasks with existing PRs, I'm encountering difficulty in linking a PR with tasks that I manually create in a GitHub Project. The issue arises because I typically plan tasks ahead of time, before the corresponding PRs are created. As a result, when I create tasks in the project's To-Do category, there's no PR available to link them to at that moment. Consequently, I end up having to delete those tasks, as I have to manually link Project with newly created PRs which creates a new task in the project (leading to duplicacy).

So, please let me know if I can link a PR directly with a Task in Github Projects interface itself.

![image](https://github.com/community/community/assets/142862928/2b1a0395-f9b2-4bda-b8e1-22915c4dc1e5)

![image](https://github.com/community/community/assets/142862928/d7c4e477-5131-427c-9701-2622d63c8166)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by yashsongara101** [comment]
@rikhiltaneja Have you found any workaround for this?
